data/brands/shop/alcohol.json

```
@@ -624,16 +624,16 @@
       }
     },
     {
-      "displayName": "Красное & Белое",
+      "displayName": "Красное&Белое",
       "id": "redandwhite-b9f2fe",
       "locationSet": {"include": ["ru"]},
       "matchNames": ["красное и белое"],
       "tags": {
-        "brand": "Красное & Белое",
+        "brand": "Красное&Белое",
         "brand:en": "Red&White",
         "brand:wikidata": "Q24933790",
-        "brand:wikipedia": "en:Krasnoe & Beloe",
-        "name": "Красное & Белое",
+        "brand:wikipedia": "en:Krasnoe&Beloe",
```

We're probably going to stop adding the `brand:wikipedia` tags anyway, but for now we should link to the Russian Wikipedia, and both the English and Russian Wikipedias pages for this brand still have spaces in their links.

```suggestion
        "brand:wikipedia": "ru:Красное и Белое",
```

---

> 

I've fix Russian Wilipedia and Wikidata. But can't fix engWikipedia — cant found «Rename button»

- https://ru.wikipedia.org/wiki/%D0%9A%D1%80%D0%B0%D1%81%D0%BD%D0%BE%D0%B5%26%D0%91%D0%B5%D0%BB%D0%BE%D0%B5
- https://www.wikidata.org/wiki/Q24933790

---

Thanks for fixing!